docker-compose.yml

```diff
@@ -6,6 +6,6 @@ services:
     container_name: jc-restapi
     image: kellybrazil/jc-restapi
     ports:
-      - "${JC_APP_PORT}:${JC_APP_PORT}"
+      - "${JC_APP_PORT:-8000}:8000"
```

Comment:
Not sure I understand this syntax. What is it doing?

Reply:
On the left side of the colon: specify the external port in the `JC_APP_PORT` variable. If it is not defined, port `8000` will be used

After the colon: Set default external port `8000` if `JC_APP_PORT` is not defined.

Reply:
Gotcha - I was weirded out by the bash-ism in the YAML file. Wasn't sure how the variable substitution was working. Looks like it uses rules similar to bash: https://docs.docker.com/reference/compose-file/interpolation/